website/pages/home/style.css

```diff
@@ -28,4 +28,13 @@
       }
     }
   }
+
+  /* HACK: Override variant-tertiary-neutral's color rules */
+  & .g-text-split .g-btn.variant-tertiary-neutral {
+    color: var(--nomad);
+
+    & svg path {
+      stroke: var(--nomad);
+    }
+  }
```

Comment:
`<Button />` doesn't override its colors with the brand when tertiary it looks like, so I hacked this in for now. This should def be addressed/considered at the component level, though.